core/jobs/batch_jobs/translation_opportunity_backfill_jobs.py

```
@@ -120,6 +127,25 @@ def _create_translation_opportunity(
+                for state in exp.states.values():
+                    pending_contents = (
+                        state.get_all_contents_which_need_translations(
+                            entity_translation
+                        ).values()
+                    )
+                    for content in pending_contents:
+                        reasons.add(content.status.value)
+                if reasons:
+                    translation_missing_reasons[lang_code] = sorted(
+                        list(reasons)
+                    )
```

Ruff (0.16.2)

[warning] Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)